iobio/clin.iobio
# Add gene drug interaction

This PR shows drug information for reviewed variants on the findings page. 
To test: 

- Load clin.iobio with demo data 
- Go to "review variants" 
- Select the variant "SCN8A" and review it. 
- Go to "report findings"   
- Click "Drugs for SCN8A" button 
- Drug information is shown in a dialog. 
- Drugs for gene button is disabled when no drug information is available.

Test: http://dev.clin.iobio.io/
